<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelectionWidget</class>
 <widget class="QWidget" name="ColorSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="r_color_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QLabel" name="g_color_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QLabel" name="b_color_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="r_color_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="b_color_lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="g_color_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="alpha_color_lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="alpha_color_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
